tpl/general/settings.tpl.php

```diff
@@ -195,12 +195,6 @@
 				⚠️ <?php echo __( 'Notice', 'litespeed-cache' ); ?>: <?php echo sprintf( __( 'You need to turn %s on and finish all WebP generation to get maximum result.', 'litespeed-cache' ),  '<code>' . Lang::title( Base::O_IMG_OPTM_WEBP ) . '</code>' ); ?>
 				</font>
 				<?php endif; ?>
-
-				<?php if ( ! $this->conf( Base::O_IMG_OPTM_WEBP_REPLACE ) ) : ?>
-				<br /><font class="litespeed-danger litespeed-left10">
-				⚠️ <?php echo __( 'Notice', 'litespeed-cache' ); ?>: <?php echo sprintf( __( 'You need to turn %s on to get maximum result.', 'litespeed-cache' ),  '<code>' . Lang::title( Base::O_IMG_OPTM_WEBP_REPLACE ) . '</code>' ); ?>
-				</font>
-				<?php endif; ?>
```

Comment:
I think we still need to give this notice

Reply:
The notice is still given immediately above.